src/libraries/System.Collections.Immutable/ref/System.Collections.Immutable.cs

```diff
@@ -702,8 +720,14 @@ public sealed partial class Builder : System.Collections.Generic.ICollection<T>,
             public int LastIndexOf(T item, int startIndex, int count) { throw null; }
             public int LastIndexOf(T item, int startIndex, int count, System.Collections.Generic.IEqualityComparer<T>? equalityComparer) { throw null; }
             public bool Remove(T item) { throw null; }
+            public void Remove(T value, IEqualityComparer<T>? equalityComparer) { throw null; }
```

Comment:
Similar to the above, cc @terrajobst 
```suggestion
            public bool Remove(T item, IEqualityComparer<T>? equalityComparer) { throw null; }
```

Reply:
@RaymondHuy could you please update the return type? It should match that of the existing overload.

Reply:
Thanks @eiriktsarpalis  I have updated the PR.